bytecomp/translcore.ml

```diff
@@ -50,6 +50,9 @@ let prim_fresh_oo_id =
   Pccall (Primitive.simple ~name:"caml_fresh_oo_id" ~arity:1 ~alloc:false)
 
 let transl_extension_constructor env path ext =
+  let path =
+    Stdlib.Option.map (Printtyp.rewrite_double_underscore_paths env) path
```

Comment:
Where does Stdlib.Option come from?   I don't see it in the PR.  (And it would be better to not depend on the bootstrapped compiled for the compiler code base in the PR itself.)

Reply:
This `Stdlib.Option` module comes from `misc.mli`: [Misc.Stdlib.Option](https://github.com/ocaml/ocaml/blob/trunk/utils/misc.mli#L83) .